x-pack/plugins/alerting/server/routes/rule/apis/aggregate/transforms/format_default_aggregation_result/v1.ts

```diff
@@ -4,96 +4,13 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import type { AggregationsAggregationContainer } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
-import {
-  RuleExecutionStatusValues,
-  RuleLastRunOutcomeValues,
-  RuleAggregationFormattedResult,
-} from './rule';
 
-export interface DefaultRuleAggregationResult {
-  status: {
-    buckets: Array<{
-      key: string;
-      doc_count: number;
-    }>;
-  };
-  outcome: {
-    buckets: Array<{
-      key: string;
-      doc_count: number;
-    }>;
-  };
-  muted: {
-    buckets: Array<{
-      key: number;
-      key_as_string: string;
-      doc_count: number;
-    }>;
-  };
-  enabled: {
-    buckets: Array<{
-      key: number;
-      key_as_string: string;
-      doc_count: number;
-    }>;
-  };
-  snoozed: {
-    count: {
-      doc_count: number;
-    };
-  };
-  tags: {
-    buckets: Array<{
-      key: string;
-      doc_count: number;
-    }>;
-  };
-}
-
-interface GetDefaultRuleAggregationParams {
-  maxTags?: number;
-}
-
-export const getDefaultRuleAggregation = (
-  params?: GetDefaultRuleAggregationParams
-): Record<string, AggregationsAggregationContainer> => {
-  const { maxTags = 50 } = params || {};
-  return {
-    status: {
-      terms: { field: 'alert.attributes.executionStatus.status' },
-    },
-    outcome: {
-      terms: { field: 'alert.attributes.lastRun.outcome' },
-    },
-    enabled: {
-      terms: { field: 'alert.attributes.enabled' },
-    },
-    muted: {
-      terms: { field: 'alert.attributes.muteAll' },
-    },
-    tags: {
-      terms: { field: 'alert.attributes.tags', order: { _key: 'asc' }, size: maxTags },
-    },
-    snoozed: {
-      nested: {
-        path: 'alert.attributes.snoozeSchedule',
-      },
-      aggs: {
-        count: {
-          filter: {
-            exists: {
-              field: 'alert.attributes.snoozeSchedule.duration',
-            },
-          },
-        },
-      },
-    },
-  };
-};
+import { RuleAggregationFormattedResult } from '../../../../../../../common/routes/rule/apis/aggregate/types/latest';
+import { RuleExecutionStatusValues, RuleLastRunOutcomeValues } from '../../../../../../../common';
+import { DefaultRuleAggregationResultV1 } from '../../types';
 
 export const formatDefaultAggregationResult = (
-  aggregations: DefaultRuleAggregationResult
+  aggregations: DefaultRuleAggregationResultV1
```

Comment:
I dont think we need to version this format function. Since the return type of this gets transformed to `AggregateRulesResponse` according to https://github.com/elastic/kibana/pull/164284#discussion_r1323648080.

If we wanted to version this, the return type has to be a versioned return type. However if we go with the comment above, `RuleAggregationFormattedResult` is no longer a versioned this. So it wouldn't make sense for this to be versioned.